pkgs/tools/networking/curl/default.nix

```diff
@@ -59,6 +59,11 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-b+oqrGpGEPvQQAr7C83b5yWKZMY/H2jlhV68DGWXEM0=";
   };
 
+  patches = lib.optionals rustlsSupport [
+    # https://github.com/curl/curl/issues/13200 and https://github.com/curl/curl/issues/13248
+    ./rustls-build-fix.patch
+  ];
```

Comment:
I don't understand why yet, but if I remove the `lib.optionals rustlsSupport` gating on this `nix-build -A rustls-ffi.tests` appears to work.

That makes _some_ sense in that the "configure: error: TLS not detected" error you observe (and that I can reproduce) would be what I would expect to happen if you used a pkg-config build without the fixes from the patches.

Reply:
Yeah, nevermind. This was totally not a thing. Sorry for the noise.

Reply:
Yeah but since the patch changes the `.m4` files we need to regenerate the files (too much autotools lately :sweat_smile: ).